extension/js/common/platform/catch.ts

```diff
@@ -193,7 +193,7 @@ export class Catch {
     } else if (origin === 'https://flowcrypt.com') {
       env = 'web:prod';
     } else if (origin === 'https://mail.google.com') {
-      env = 'ex:script:gmail';
+      env = 'ex:s:gmail';
     }
```

Comment:
Hi @sosnovsky - Do you think it's necessary to add comments explaining "env" if it stands for "script"?

Reply:
I think it's ok to leave it without comment, as currently we use this property only for differentiating environments where exception happened.

Reply:
Oh, okay - thanks!